wiki/magnitude

```diff
@@ -0,0 +1,58 @@
+--- 
+title: Magnitude
+layout: wiki
+---
+
+# Introduction
+
+The Gridcoin network encompasses multiple types of BOINC projects, each one with a different methodology for calculating credit. These differences are reconciled through an innovative mechanism called Magnitude, which subsequently determines reward calculations.
+
+# Methodology
+
+The Gridcoin network has a Total Magnitude of 115,000 that is evenly distributed across all whitelisted projects. As there are currently 16 whitelisted projects, this results in a Project Magnitude of 7,187.5. This Project Magnitude is distributed among its Gridcoin crunchers in proportion to their RAC. Lastly, each Cruncher's Magnitude is calculated by adding the total sum of their magnitudes across all whitelisted projects. This calculation is illustrated through the examples below. 
+
+# Impact
+
+By distributing the Total Magnitude of 115,000 equally across all projects, the Gridcoin Network ensures that less popular projects would provide higher returns for crunchers. This creates an incentive for crunching power to be distributed evenly across the whitelisted projects, thereby ensuring that all whitelisted projects benefit equally from the network.
+
+# Rewards Calculation
+
+GRC rewards for crunching is determined per superblock, which have a current rate of c. 1 superblock per day, and can be calculated from a cruncher's magnitude according to the following formula:
+
+- GRC per Superblock = Cruncher's Magnitude / 4
+
+These pending rewards are updated by the second in the GRC wallet.
```

Comment:
The issue is `GRC per Superblock` and `determined per superblock` make it sound like research rewards are calculated and updated on the superblocks instead of just magnitude. Here's an alternative phrasing:

```suggestion
Gridcoin rewards for crunching are determined by magnitude. Magnitude updates every superblock which happen nearly once per 24 hours. The reward can be calculated from a cruncher's magnitude according to the following formula:

- Gridcoin per Day = Cruncher's Magnitude / 4

The pending rewards themselves are updated by the second in the Gridcoin wallet.
```